client/my-sites/stats/activity-log-day/placeholder.js

```diff
@@ -0,0 +1,26 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+
+const component = (
+	<FoldableCard
+		className="activity-log-day__placeholder"
+		header={
+			<div>
+				<div className="activity-log-day__day" />
+				<div className="activity-log-day__events" />
+			</div>
+		}
+	/>
+);
+
+export default function ActivityLogDayPlaceholder() {
```

Comment:
Any good reason to have a function returning the component tree instead of:

```
const ActivityLogDayPlaceholder = () => (
	...
);

export default ActivityLogDayPlaceholder;
```

or even just:

```
export default () => (
	...
);
```